windows.bat

```diff
@@ -0,0 +1,15 @@
+@rem add nmake command to cmd
+set vstoolsroot=%VS140COMNTOOLS%
```

Comment:
Aslında benimde istediğim bir şey değil ama sanırım nmake'in standalone versiyonu bulunup projeye eklenebilir. Yukarıda bahsettiğim prod-dev ayarına göre silinebilir.